mmv1/products/compute/HaVpnGateway.yaml

```diff
@@ -135,6 +135,17 @@ properties:
     values:
       - :IPV4_ONLY
       - :IPV4_IPV6
+      - :IPV6_ONLY
+    immutable: true
+    custom_flatten: 'templates/terraform/custom_flatten/default_if_empty.erb'
+  - !ruby/object:Api::Type::Enum
+    name: 'gatewayIpVersion'
+    description: |
+      The IP family of the gateway IPs for the HA-VPN gateway interfaces. If not specified, IPV4 will be used.
+    default_value: :IPV4
```

Comment:
with this field having a default value, let's add a test case that ensures that the default value is in fact `IPV4` to prevent future changes from breaking the test.

Reply:
Added a check for default value. Removed manually setting the values in the config string.